orderedmap.go

```diff
@@ -40,6 +40,15 @@ func (om *OrderedMap) Get(key interface{}) (interface{}, bool) {
 	return nil, false
 }
 
+// GetPair looks for the given key, and returns the pair associated with it,
+// or nil if not found. The boolean it returns says whether the key is present in the map.
+func (om *OrderedMap) GetPair(key interface{}) (*Pair, bool) {
```

Comment:
I don't see the point of returning a bool on top of the `*Pair` itself, since the bool will be true if and only if the pointer is not nil, so returning the pointer alone is sufficient?

Reply:
then it is very simple
```go
	return om.pairs[key]
```